experimental/material_3_demo/analysis_options.yaml

```diff
@@ -0,0 +1,29 @@
+# This file configures the analyzer, which statically analyzes Dart code to
```

Comment:
We have centralised the lint configuration for this repository. Please include the confit from the root of the repository